### DISCUSSION TITLE
How to create new workspace or app?

### DISCUSSION THREAD

**Discussion by paulmuenzner** [vercel/turborepo#2573] [Help] [answered]
Hello,
I really wonder why there is no detailed description at all describing how to create a new app (react with own webpack, next, ...) with some valid example commands and explanation on turbo website. Same for generating libraries (ts files, tsx components, ...). Also mentioning what to take care of all and which options are available. Same for moving, renaming or deleting workspace librabries.

Can somebody please help me to find this? I cannot believe it is not on their homepage: https://turbo.build/repo/docs
If yes, it probably should be much more easy to find.
I expected it to find under https://turbo.build/repo/docs/getting-started/create-new

Everybody who is new to turborepo might be easily upset at the very beginning.

Could somebody provide help please?

Kind regards

**Comment by nathanhammond** [comment, marked as answer]
Everything you mentioned is covered on the page you linked, and in the Monorepo Handbook:
￼
https://turbo.build/repo/docs/handbook

**Reply by paulmuenzner** [reply]
Thanks, more or less. But better than nothing.
